site/contact.html

```diff
@@ -9,7 +9,10 @@
       <div class="col-xs-12 col-lg-7 centered">
         <h1>Get in touch</h1>
         <p>
-          Send us your questions about hosting, support or professional services.
+          Send us your questions about hosting, support plans or professional services.
+        </p>
+        <p>
+          If you are looking for technical support, please open a support ticket from your <a href="https://account.fusionauth.io/account/support/">account page</a> if you have a plan with support. If you are on a plan without support (Community or Developer) please <a href='https://fusionauth.io/community/forum/'>post your question in the forum</a>.
```

Comment:
Can you use double quotes for the second `href` attribute?

Reply:
Done